streaming/streaming.go

```diff
@@ -100,11 +104,37 @@ func VerifyAndUpdateCfg(streamingCfg *StreamingCfg, dbName string) error {
 	if dfCfg.JobName == "" {
 		// Update names to have more info like dbname.
 		jobName, err := utils.GenerateName("hb-dataflow-" + dbName)
+		jobName = strings.Replace(jobName, "_", "-", -1)
 		if err != nil {
 			return fmt.Errorf("error generating stream name: %v", err)
 		}
 		streamingCfg.DataflowCfg.JobName = jobName
 	}
+
+	filePath := streamingCfg.TmpDir
+	if filePath[len(filePath)-1] != '/' {
```

Comment:
This could break if the length of streamingCfg.TmpDir is 0 , would be better to handle that case as well?

Reply:
Good catch, fixed!